crates/holochain_p2p/src/types/event.rs

```diff
@@ -4,44 +4,6 @@
 use crate::*;
 use holochain_zome_types::signature::Signature;
 
-/// The data required for a get request.
-#[derive(Default, Debug, serde::Serialize, serde::Deserialize, Clone)]
-pub enum GetRequest {
-    /// Get all the integrated data.
-    #[default]
-    All,
-    /// Get only the integrated content.
-    Content,
-    /// Get only the metadata.
-    /// If you already have the content this is all you need.
-    Metadata,
```

Comment:
That's an interesting idea, but you can't check hashes and signatures without the content, so I can't see how it would work in practice...